src/app/profile/profile.component.js

```diff
@@ -6,8 +6,8 @@ import profileService from 'common/services/api/profile.service';
 import loadingOverlay from 'common/components/loadingOverlay/loadingOverlay.component';
 import sessionEnforcerService, {EnforcerCallbacks, EnforcerModes} from 'common/services/session/sessionEnforcer.service';
 import {Roles} from 'common/services/session/session.service';
-import formatAddressForTemplate from 'common/services/addressHelpers/formatAddressForTemplate';
 import showErrors from 'common/filters/showErrors.filter';
+import {Observable} from 'rxjs/Observable';
```

Comment:
Can you explicitly include `forkJoin` and `do` here?
```
import 'rxjs/add/observable/forkJoin';
import 'rxjs/add/operator/map';
```
I think it works cuz some other file includes them but they might not always be available in tests or if the files that include this component remove those import.